### DISCUSSION TITLE
How to delete account

### DISCUSSION THREAD

**Discussion by GarbageCollectionInC** [mastodon/mastodon#27160] [Q&A] [answered]
Sorry, I want to delete my account but how? The instructions I find on internet do not work; and I have no idea how to proceed. My account is on mstdn.social. Any help is appreciated.

**Comment by StrangeMinds1960** [comment, marked as answer]
I'm guessing you've seen this already, but I'm going to post it anyway

https://docs.joinmastodon.org/user/moving/#delete

**Reply by GarbageCollectionInC** [reply]
Thanks , I will try
